terraform/common.tf

```diff
@@ -31,7 +31,7 @@ variable "sample_app_image" {
 }
 
 variable "aoc_image_repo" {
-  default = "611364707713.dkr.ecr.us-west-2.amazonaws.com/aws/aws-otel-collector"
+  default = "public.ecr.aws/aws-observability/aws-otel-collector"
```

Comment:
One of the reason I changed this is because the `private repo image does not have the latest tag` except the public one. Therefore, we could change this in this PR or another PR if that make sense.